src/model/parser/flow_node_parsers/activity_parsers/user_task_parser.ts

```diff
@@ -0,0 +1,181 @@
+    return [];
+  }
+
+  for (const userTaskRaw of userTasksRaw) {
+    const userTask: Model.Activities.UserTask = createActivityInstance(userTaskRaw, Model.Activities.UserTask);
+
+    userTask.assignee = userTaskRaw[BpmnTags.CamundaProperty.Assignee];
+    userTask.candidateUsers = userTaskRaw[BpmnTags.CamundaProperty.CandidateUsers];
+    userTask.candidateGroups = userTaskRaw[BpmnTags.CamundaProperty.CandidateGroups];
+    userTask.dueDate = parseDate(userTaskRaw[BpmnTags.CamundaProperty.DueDate]);
+    userTask.followUpDate = parseDate(userTaskRaw[BpmnTags.CamundaProperty.FollowupDate]);
+    userTask.formFields = parseFormFields(userTaskRaw);
+    userTask.preferredControl = getPreferredControlForUserTask(userTaskRaw);
+    userTask.description = getDescriptionForUserTask(userTaskRaw);
+    userTask.finishedMessage = getFinishedMessageForUserTask(userTaskRaw);
+
+    userTasks.push(userTask);
+  }
+
+  function parseFormFields(userTaskRaw: any): Array<Model.Activities.Types.UserTaskFormField> {
+
+    const extensionElements: any = userTaskRaw[BpmnTags.FlowElementProperty.ExtensionElements];
+    if (!extensionElements) {
+      return [];
+    }
+
+    const formDataRaw: any = extensionElements[BpmnTags.CamundaProperty.FormData];
+    if (!formDataRaw) {
+      return [];
+    }
+
+    const formFieldsRaw: any = getModelPropertyAsArray(formDataRaw, BpmnTags.CamundaProperty.FormField);
+    if (!formFieldsRaw) {
+      return [];
+    }
+
+    const formFields: Array<Model.Activities.Types.UserTaskFormField> = [];
+
+    for (const formFieldRaw of formFieldsRaw) {
+      const formField: Model.Activities.Types.UserTaskFormField = parseFormField(formFieldRaw);
+      formFields.push(formField);
+    }
+
+    return formFields;
+  }
+
+  function parseFormField(formFieldRaw: any): Model.Activities.Types.UserTaskFormField {
+
+    const formField: Model.Activities.Types.UserTaskFormField = new Model.Activities.Types.UserTaskFormField();
+
+    formField.id = formFieldRaw.id;
+    formField.label = formFieldRaw.label;
+    formField.type = formFieldRaw.type;
+    formField.defaultValue = formFieldRaw.defaultValue;
+    formField.preferredControl = formFieldRaw.preferredControl;
+
+    if (formField.type === 'enum') {
+      const rawValues: Array<any> = getModelPropertyAsArray(formFieldRaw, BpmnTags.CamundaProperty.Value);
+
+      const valueMapper: any = (enumValueRaw: any): Model.Activities.Types.FormFieldEnumValue => {
+        const enumValue: Model.Activities.Types.FormFieldEnumValue = new Model.Activities.Types.FormFieldEnumValue();
+        enumValue.id = enumValueRaw.id;
+        enumValue.name = enumValueRaw.name;
+
+        return enumValue;
+      };
+      formField.enumValues = rawValues ? rawValues.map(valueMapper) : [];
+    }
+
+    return formField;
+  }
+
+  function parseDate(value: string): Date {
+
+    if (!value || value.length === 0 || !moment(value, 'YYYY-MM-DDTHH:mm:ss', true).isValid()) {
```

Comment:
So I guess this is checking if the value is a valid date?
If so:

```suggestion
    const isValidDate: boolean = !value || value.length === 0 || !moment(value, 'YYYY-MM-DDTHH:mm:ss', true).isValid();
    if (isValidDate) {
```